lib/src/test/java/io/ably/lib/test/realtime/RealtimePresenceTest.java

```diff
@@ -302,6 +303,10 @@ public void enter_leave_simple() {
 			leaveComplete.waitFor();
 			assertTrue("Verify leave callback called on completion", leaveComplete.success);
 
+			try {
+				assertEquals("Verify item is removed from the presence map", client1Channel.presence.get(testClientId1, false).length, 0);
```

Comment:
Actually I found `TP3h` spec and I'll move the `memberKey` function to `PresenceMessage` and make it public